client/components/select-vertical/index.tsx

```diff
@@ -57,6 +64,7 @@ const SelectVertical: React.FC< Props > = ( { defaultVertical, isSkipSynonyms, o
 				suggestions={
 					! isDebouncing ? mapManySiteVerticalsResponseToVertical( suggestions || [] ) : []
 				}
+				featuredSuggestions={ mapManySiteVerticalsResponseToVertical( featured || [] ) }
```

Comment:
Could we just use `suggestions`? That being said, if the `searchTerm` is empty, we use the feature suggestions

Reply:
Sounds good to me 🙂